app/javascript/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js

```diff
@@ -68,9 +68,13 @@ function TermForm(props) {
       },
     };
 
+    // ALL CAPS enum value per graphql API
+    const termTypeUcase = termType.toUpperCase();
```

Comment:
If you change `const types = ['external', 'local', 'temporary'];` earlier in the file to `const types = ['EXTERNAL', 'LOCAL', 'TEMPORARY'];`, can we eliminate the need for `termTypeUcase`?